app/views/admins/sessions/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2>Log in as Admin</h2>
```

Comment:
Following on from my comments about the commented out controller sitting alone in the `Admins` namespace, I think this view should be in the `admin` (singular) folder with the rest of the admin-related views, and where the `Admin::SessionsController` (singular) can hopefully find it.